Sources/Public/Animation/LottieView.swift

```diff
@@ -16,28 +15,44 @@ public struct LottieView<Placeholder: View>: UIViewConfiguringSwiftUIView {
   public init(animation: LottieAnimation?) where Placeholder == EmptyView {
     _animationSource = State(initialValue: animation.map(LottieAnimationSource.lottieAnimation))
     placeholder = nil
+    loadAnimationTrigger = .constant(LoadReference.constantSource)
   }
 
   /// Creates a `LottieView` that displays the given `DotLottieFile`
   public init(dotLottieFile: DotLottieFile?) where Placeholder == EmptyView {
     _animationSource = State(initialValue: dotLottieFile.map(LottieAnimationSource.dotLottieFile))
     placeholder = nil
+    loadAnimationTrigger = .constant(LoadReference.constantSource)
   }
 
   /// Creates a `LottieView` that asynchronously loads and displays the given `LottieAnimation`.
   /// The `loadAnimation` closure is called exactly once in `onAppear`.
-  public init(_ loadAnimation: @escaping () async throws -> LottieAnimation?) where Placeholder == EmptyView {
-    self.init(loadAnimation, placeholder: EmptyView.init)
+  /// - Parameters:
+  ///   - loadAnimationTrigger: `Binding` that  triggers a new call to `loadAnimation` when a new value is propagated
+  ///   - loadAnimation: Closure that provides an animation asynchronously
+  public init(
+    loadAnimationTrigger: Binding<AnyHashable> = .constant(LoadReference.constantSource),
```

Comment:
Instead of adding this an an argument to all of the `init` values, could we add a builder function for this (e.g. `func loadAnimationTrigger(_ binding: Binding<AnyHashable>) -> Self`)? 

This is optional and won't be used in all use cases. Anything optional seems better to provide using builder functions rather than in each individual `init`. 

Then we can just mention this in the doc comment of all of the `loadAnimation` initializers, like "By default the `loadAnimation` closure is called exactly once in `onAppear`. To trigger the closure to be called again at a later time, configure `.loadAnimationTrigger($reloadAnimation)` with a binding.